buildspec.yml

```diff
@@ -31,35 +32,52 @@ phases:
           GITHUB_BRANCH="$(git branch -a --contains HEAD | sed -n 2p | awk '{ printf $1 }')";
           export GITHUB_BRANCH=${GITHUB_BRANCH#remotes/origin/};
         fi
-      - COMMIT_HASH=$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | cut -c 1-7)
-      - export IMAGE_TAG="${GITHUB_BRANCH}.${COMMIT_HASH}"
-      - echo "GITHUB_BRANCH=${GITHUB_BRANCH} COMMIT_HASH=${COMMIT_HASH} IMAGE_TAG=${IMAGE_TAG}"
+      - export GITHUB_COMMIT=$(echo $CODEBUILD_RESOLVED_SOURCE_VERSION | cut -c 1-7)
+      - export GITHUB_TAG="$(git describe --tags 2>/dev/null)"
+      - export IMAGE_TAG="${GITHUB_BRANCH}.${GITHUB_COMMIT}"
```

Comment:
IMAGE_TAG is not used anymore and should be removed

Reply:
Thanks